src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -101,6 +104,40 @@ void createStationWithDuplicateName() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        //given
+        //지하철역_생성_목록
+        List<String> names = new ArrayList<>();
+        names.add("신림역");
+        names.add("강남역");
+
+        // when
+        // 지하철역_생성
+        names.stream().forEach(s -> createStation(s));
+
+        //then
+        // 지하철_노선_조회
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract();
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList(".", StationResponse.class).size()).isEqualTo(2);
```

Comment:
hasSize 를 사용해 볼 수도 있을 것 같아요 😄 
```suggestion
        assertThat(response.jsonPath().getList(".", StationResponse.class))..hasSize(2);
```